components/list/list-item-mixin.js

```diff
@@ -394,12 +396,12 @@ export const ListItemMixin = superclass => class extends composeMixins(
 				padding-left: 12px;
 				padding-right: 0;
 			}
-			:host([expandable]) .d2l-list-item-color-outer {
-				padding-right: 6px;
+			.d2l-list-item-color-outer + .d2l-list-expand-collapse {
+				margin-left: -6px;
```

Comment:
This wasn't behaving as desired when expanded and a sibling had a color indicator

Reply:
So many scenarios!